app/all-plants/[plantId]/style.ts

```diff
@@ -0,0 +1,81 @@
+import styled from 'styled-components';
+import COLORS from '@/styles/colors';
+
+export const Container = styled.div`
+  padding: 20px;
+`;
```

Comment:
we should remove the padding from Container and instead include `margin: 24px;` in `Content`